src/graphql.ts

```diff
@@ -73,6 +73,17 @@ interface ParsedQueryPayload {
   operationName: string | undefined
 }
 
+interface ParsedQueryAndTypePayload {
```

Comment:
Can we use `GraphQLRequestParsedResult` instead? An arbitrary GraphQL operation can also have variables. Once we add those, two interfaces look identical.

Reply:
Should be resolved in new commit